app/api/views/briefs.py

```diff
@@ -411,6 +411,12 @@ def post_brief_response(brief_id):
         if 'essentialRequirements' in e.message and e.message['essentialRequirements'] == 'answer_required':
             message = "Essential requirements must be completed"
             del e.message['essentialRequirements']
+        if 'attachedDocumentURL' in e.message:
+            if e.message['attachedDocumentURL'] == 'answer_required':
+                message = "Uploaded documents are missing"
```

Comment:
This message might be misleading.  Were the documents uploaded and then can't be found?  Maybe something like:
> Documents must be uploaded

Or can we be more specific about which document is missing?

Reply:
fixed